data/os/RedHat.yaml

```diff
@@ -0,0 +1,2 @@
+---
+nomad::bin_dir: /bin
```

Comment:
On Red Hat `/bin` is a symlink to `/usr/bin`, right? So can't we just unify on `/usr/bin`?

Reply:
you are absolutely right again, pushed a fix